app/javascript/components/venue/VenueEdit.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import Axios from 'axios';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+
+export default class VenueEdit extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      venue: { address: '', description: '', people_capacity: '' }
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange = venue => {
+    const params = this.state;
+    params['venue'][venue.target.name] = venue.target.value;
+    this.setState({
+      params
+    });
+  };
+
+  handleSubmit = venue => {
+    venue.preventDefault();
+    let headers = {};
+    if (sessionStorage.user) {
+      headers = JSON.parse(sessionStorage.user);
+    }
+    headers['X-CSRF-Token'] = document
+      .querySelector('meta[name=\'csrf-token\']')
+      .getAttribute('content');
+    Axios.patch(
+      `/api/v1/venues/${this.props.match.params.id}`,
+      this.state.venue,
```

Comment:
Rewrite please
{ venue: this.state.venue }

You can check the changes params in controller

Reply:
rewritten